lib/serviceBusMessage.ts

```diff
@@ -897,4 +897,30 @@ export class ServiceBusMessage implements ReceivedMessage {
       throw new Error(`Cannot find the receiver with name '${this.delivery.link.name}'.`);
     }
   }
+
+  /**
+   * Creates a clone of the current message to allow it to be re-sent to the queue
+   * @returns ServiceBusMessage
+   */
+  clone(): SendableMessageInfo {
+    // We are returning a SendableMessageInfo object because that object can then be sent to ServiceBus
```

Comment:
I would merge the information in this comment into the jsdocs above

Reply:
Yup. Putting that information in js docs would be useful.

Reply:
Done